app/views/patients/new.html.erb

```diff
@@ -0,0 +1,37 @@
+<%= link_to("Go back to list", patients_path, :class => 'back-link') %>
+
+<div class="subjects new">
+  <h2>Create Patient</h2>
+
+  <%= form_for(@patient) do |f| %>
+
```

Comment:
basically remove all blank lines at the start or end of an indent block - I'll stop these comments too. just apply everywhere